src/main/java/uk/gov/companieshouse/api/accounts/validation/PreviousPeriodValidator.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.api.accounts.validation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.model.rest.FixedAssets;
+import uk.gov.companieshouse.api.accounts.model.rest.PreviousPeriod;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+
+import javax.validation.Valid;
+
+@Component
+public class PreviousPeriodValidator extends BaseValidator {
+
+    String PREVIOUS_PERIOD_PATH = "$.previous_period";
```

Comment:
Can you add the accessors for these variables.

Reply:
do we need accessors for these? We don't have them in the current period validator and they are only used within this class

Reply:
I presume you are declaring a constant given it is declared in all upper case, in which case we should be following our development standards in this regards. It should be prefixed with `private static final` or `public static final`. I suggest we change it in the CurrentPeriodValidator as well.